### PR TITLE
feat: back button on create wallet page

### LINKED ISSUE #550: Back button on create wallet page
## Story

**As a** user
**I want** top be able to go back from create wallet
**So that** I can import a wallet instead

## Acceptance Criteria

- [ ] I can go back from the create a wallet page

## Tasks

- [ ] Detailed sketch
- [ ] Visual design
- [ ] Frontend tasks:
    - [ ] Implement UI
    - [ ] Integrate with backend
    - [ ] Write unit tests
- [ ] Backend tasks:
    - [ ] TBC
    - [ ] Write unit tests
- [ ] Write end to end tests
- [ ] Code review
- [ ] Team verify acceptance criteria met
- [ ] Exploratory QA testing 
- [ ] Capture bugs and new edge cases in tickets (this one or new TBD) 
- [ ] Write automated regression tests for the A/Cs (expanding on A/Cs where possible) 
- [ ] Add story and A/Cs to spec doc 
- [ ] Review with stakeholders
- [ ] Release

### PR DESCRIPTION
**Pull request by dexturr**
Closes #550

### CODE REVIEW

File: web-extension/lib/setup-listeners.js [hunk 1]
```diff
@@ -11,22 +11,22 @@ export const createConnectionHandler = (clientPorts, popupPorts, interactor) =>
...
-    await Promise.allSettled([
-      networks.set(id, {
-        name: config.network.name,
-        rest: config.network.rest,
-        explorer: config.network.explorer
-      }),
-      settings.set('selectedNetwork', id),
-      settings.set('autoOpen', true)
-    ])
+  // if (reason === 'install') {
```
Comment (dexturr) [inline web-extension/lib/setup-listeners.js:14]: REMOVE

**Review by dalebennett1992 — APPROVED** [review]
